docs/user_guide.md

```diff
@@ -1179,6 +1182,9 @@ be compared to each other:
 CodeChecker cmd diff -p 8001 --basename my_project --newname my_new_checkin --new
 ~~~~~~~~~~~~~~~~~~~~~
 
+There is opportunity to compare a run to multiple baselines. You can simply
+provide a regular expression by `-b` flag which covers the required run names.
```

Comment:
insert here a link that we specifically support this syntax https://docs.python.org/2/library/re.html